### ISSUE TITLE
Cannot paste Clipboard text into Uno running within a browser

### ISSUE THREAD

**Issue by c-npolyak** [unoplatform/uno#12922]
### Current behavior

I am using Uno.WebAssembly 4.7.37. Here is the code 

     DataPackageView dataPackageView =
                Windows.ApplicationModel.DataTransfer.Clipboard.GetContent();
     
     if (dataPackageView.Contains("Text"))
     {    
          var text = await dataPackageView.GetTextAsync();
     }
      
When I run it in the VS debugger the line ```var text = await dataPackageView.GetTextAsync();``` is hit but never returns. Instead uno opens an empty file AsyncTaskMethodBuilderT.cs from Uno.DotnetRuntime.WebAssembly.

I tried to handle the Completed event of the ```IAsyncOperation<string>``` and pulled the information about the error from 
AsycnInfo object, it is InvalidOperationException with message

`Failed to asynchronously load the data of id 'Text'`

 and the stack is 

```
at Windows.ApplicationModel.DataTransfer.DataPackageView.<GetDataAsync>d__26`1[[System.String, System.Private.CoreLib, Version=7.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].MoveNext() in /home/vsts/work/1/s/src/Uno.UWP/ApplicationModel/DataTransfer/DataPackageView.cs:line 115
at Windows.Foundation.AsyncOperation`1.<BuildTaskAsync>d__5[[System.String, System.Private.CoreLib, Version=7.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].MoveNext() in /home/vsts/work/1/s/src/Uno.Foundation/AsyncOperation.TResult.cs:line 34
```

The InvalidOperationException has TaskCanceledException as its inner Exception which was thrown from DataPackageView.cs file at line 110

### Expected behavior

It should return the text contained in the clipboard. 

### Works on UWP/WinUI

None

### Affected platforms

WebAssembly

### IDE

Visual Studio 2022

**Comment by jeromelaban** [comment]
We've made fixes in this area since Uno 4.7. Make sure to try with a more recent version of Uno, like 4.9.x and let us know if it helps. Otherwise, make sure to take a look at [the documentation here](https://platform.uno/docs/articles/features/clipboard.html).

**Comment by MartinZikmund** [comment]
Closing as fixed, feel free to reopen if happens again
